docs/reference/resources.md

```diff
@@ -2,6 +2,8 @@
 title: "Resources"
 weight: 10
 type: "docs"
+aliases:
+   - /docs/concepts/resources.md
```

Comment:
Cool, you got this one but we need to remove the `.md`. These redirect the HTML paths (vs github path)

More about aliases: https://gohugo.io/content-management/urls/#aliases

Reply:
Ah, I see, so follow up PR to change the aliases after this merges, or...? :slightly_smiling_face: 

Question: have you ever used asciidoc and would you have any interest in looking at a proof of concept for this with Hugo? It provides easier solutions for a lot of the issues we have with markdown files and it's not any more difficult to use IMHO.